probes/src/main/java/com/hazelcast/simulator/probes/probes/impl/HdrLatencyDistributionResult.java

```diff
@@ -30,9 +30,10 @@ public HdrLatencyDistributionResult combine(HdrLatencyDistributionResult other)
         if (other == null) {
             return this;
         }
-        Histogram histogramCopy = new Histogram(histogram);
-        histogramCopy.add(other.histogram);
-        return new HdrLatencyDistributionResult(histogramCopy);
+        Histogram combinedHistogram = new Histogram(histogram);
```

Comment:
This does NOT copy the content, just the parameters. So we have to call `combinedHistogram.add()` for both histograms.